package.json

```
@@ -7,7 +7,7 @@
     "dev": "babel src --out-dir dist",
     "build": "BABEL_ENV=production babel src --out-dir dist",
     "prepublishOnly": "npm run build",
-    "test": "make test",
+    "test": "./node_modules/.bin/mocha --reporter spec",
```

```suggestion
    "test": "mocha --reporter spec",
```

NPM knows how to call mocha here already.

---

You're right. I made the change in 19dfe2e3a5e5fe9df3d08284891e2e60ba96e6e6.